vendor/vendor.json

```diff
@@ -258,6 +264,12 @@
 			"version": "v12.5.0-beta",
 			"versionExact": "v12.5.0-beta"
 		},
+		{
+			"checksumSHA1": "0f2bCAIQjO9op810x5LnuZnAmM0=",
+			"path": "github.com/Azure/azure-sdk-for-go/version",
+			"revision": "1e334c402ea1460704b0263e5d188f28ad946ce1",
+			"revisionTime": "2018-02-16T17:41:56Z"
```

Comment:
(which should mean this isn't needed); although we plan to upgrade in the near future #825